### DISCUSSION TITLE
How to extract all MP4 metadata?

### DISCUSSION THREAD

**Discussion by dhoffer** [drewnoakes/metadata-extractor#658] [Q&A]
I have been testing with several MP4 files and the metadata-extractor library can extract the MP4, MP4 Video, MP4 Sound metadata directories but none of these contain the basic file metadata such as Title, Artist, Comments, Composer, Copyright, etc.  How can get this type of metadata from MP4 files.  This feature does work for QuickTime files.

**Comment by drewnoakes** [comment]
This library's support for video formats is not as rich as for images. If you have a (small) file you can share that contains such metadata, it would help. Better yet, search for standards/documentation online, break open an IDE and try to add support. We can support any questions or issues you might hit.
